### PR TITLE
change routing on express server issue #27

### PR DESCRIPTION
**Pull request by dave2000sang**

### CODE REVIEW

File: contrail-api/src/server.ts [hunk 1]
```diff
@@ -35,15 +35,16 @@ app.post("/login", (req, res) => {
       .auth()
       .verifyIdToken(token)
       .then((decodedToken) => {
-        res.sendStatus(200);
+        res.status(200).send("userLoginSuccessful");
       })
       .catch((error) => {
+        res.status(200).send("userLoginFailed")
```
Comment (notoriousmango) [inline contrail-api/src/server.ts]: use 500

File: contrail-api/src/server.ts [hunk 2]
```diff
@@ -66,21 +67,14 @@ app.post("/register", (req, res) => {
           profileImageUri: null,
           userId: userRecord.uid,
         });
+        console.log("register user complete");
+        res.status(200).send("userRegistrationSuccessful");
       })
       .catch((error) => {
         console.log("error registering user", error);
-        res.send("error: registration");
+        res.status(200).send("userRegistrationFailed");
```
Comment (notoriousmango) [inline contrail-api/src/server.ts]: use status 500

File: contrail-api/src/server.ts [hunk 3]
```diff
@@ -35,15 +35,16 @@ app.post("/login", (req, res) => {
...
         console.error(`/login - ${error}`);
       });
   }
 });
 
-app.post("/register", (req, res) => {
+app.post("/api/register", (req, res) => {
   const token = req.body;
   if (token == null) {
     res.status(400).send("Bad Request: ID Token is null");
```
Comment (a29paul) [inline contrail-api/src/server.ts]: Should be 401, Auth required

File: contrail-api/src/server.ts [hunk 4]
```diff
@@ -26,27 +26,28 @@ app.get("/", (req, res) => {
   res.sendFile(authPage);
 });
 
-app.post("/login", (req, res) => {
+app.post("/api/login", (req, res) => {
```
Comment (1298se) [inline contrail-api/src/server.ts]: lets rename the route to /api/auth because now its not only being used in login

**Pull request closed without merging**